solutions/fully-configurable/catalogValidationValues.json.template

```diff
@@ -0,0 +1,3 @@
+{
+  "ibmcloud_api_key": $VALIDATION_APIKEY
```

Comment:
We should be enabling the cluster with KMS encryption so SCC rules pass those checks.

Reply:
Done.